clang_tidy/__init__.py

```diff
@@ -6,15 +6,24 @@
 def _get_executable(name):
     return os.path.join(os.path.dirname(__file__), "data", "bin", name)
 
-def _run(name):
-    executable = _get_executable(name)
-    return subprocess.call([executable] + sys.argv[1:])
+def _run(name, *args):
+    command = [_get_executable(name)]
+    if args:
```

Comment:
Can you help me understand why this branching is needed?

Reply:
The new `args` tuple argument allows executing clang-tidy from Python (used in the test). The branch keeps the previous behavior if no argument is provided to the function.